seed/lib/merging/merging.py

```diff
@@ -84,6 +88,50 @@ def get_state_attrs(state_list):
         return get_taxlotstate_attrs(state_list)
 
 
+def _merge_geocoding_results(merged_state, state1, state2, priorities, can_attrs, ignore_merge_protection=False):
+    """
+    Geocoding results need to be handled separately since they should generally
+    "stick together". In one sense, all 4 result columns should be treated as
+    one column. Specifically, the complete geocoding results of either the new
+    state or the existing state is used - not a combination of the geocoding
+    results from each.
+    """
+    geocoding_attr_cols = [
+        'geocoding_confidence',
+        'longitude',
+        'latitude',
+        'long_lat',  # note this col shouldn't have priority set
+    ]
+
+    existing_results_empty = True
+    new_results_empty = True
+    geocoding_favor_new = True
+
+    for geocoding_col in geocoding_attr_cols:
+        existing_results_empty = existing_results_empty and can_attrs[geocoding_col][state1] is None
+        new_results_empty = new_results_empty and can_attrs[geocoding_col][state2] is None
+
+        geocoding_favor_new = geocoding_favor_new and priorities.get(geocoding_col, 'Favor New') == 'Favor New'
+
+        # Since these are handled here, remove them from canonical attributes
+        del can_attrs[geocoding_col]
+
+    # Multiple elif's here is necessary since empty checks should be first, followed by merge protection settings
+    if new_results_empty:
```

Comment:
seems like possible performance issues here... we'll cross that bridge when we get there.